src/systemctl/systemctl-show.c

```diff
@@ -419,12 +421,20 @@ static void print_status_info(
                     STRPTR_IN_SET(i->active_state, "activating")          ? i->inactive_exit_timestamp :
                                                                             i->active_exit_timestamp;
 
-        if (timestamp > 0 && timestamp < USEC_INFINITY)
+        if (timestamp > 0 && timestamp < USEC_INFINITY) {
                 printf(" since %s; %s\n",
                        FORMAT_TIMESTAMP_STYLE(timestamp, arg_timestamp_style),
                        FORMAT_TIMESTAMP_RELATIVE(timestamp));
-        else
+                if (STRPTR_IN_SET(i->active_state, "active") &&
```

Comment:
STRPTR_IN_SET() is for checking if a string match a list of strings. here you only want to compare a single one, i.e. us streq_ptr()